packages/snap-preact/components/src/components/Organisms/Facet/Facet.tsx

```diff
@@ -307,9 +334,7 @@ export const Facet = observer((properties: FacetProps): JSX.Element => {
 							ref={(e) => useA11y(e, disableCollapse ? -1 : 0)}
 							role="heading"
 							aria-level={3}
-							aria-label={`currently ${facet?.collapsed ? 'collapsed' : 'open'} ${facet.label} facet dropdown ${
-								(facet as ValueFacet).values?.length ? (facet as ValueFacet).values?.length + ' options' : ''
-							}`}
+							{...mergedLang.dropdownButton}
 						>
 							{facet?.label}
 							{!disableCollapse && (
```

Comment:
This dropdown icon would be replaced when using the `dropdownButton` value lang prop. Proposal to split the attributes and value in the `useLang` response.